documentation/manual-lobster_cpptest.md

```diff
@@ -0,0 +1,115 @@
+# Tracing to C/C++ test requirements
+
+## Setup and requirements
+
+You will need a C/C++ file or a directory containing.
+
+C/C++ files should have one of these extensions to be evaluated by this tool: `.cpp, .cc, .c or .h`
+
+## Preparing C/C++ test comments with requirements
+The test functions should be specified with one the following names: 
+        `TEST`,
+        `TEST_P`,
+        `TEST_F`,
+        `TYPED_TEST`,
+        `TYPED_TEST_P`,
+        `TYPED_TEST_SUITE`,
+        `TEST_P_INSTANCE`,
+        `TEST_F_INSTANCE`
+
+In your test you need to add comments. For example:
+
+```C++
+/**
+ * @requirement CB-#0815, CB-#0816,
+ * 				CB-#0817
+ * @requirement CB-#0818 CB-#0819
+ * 				CB-#0820
+ */
+TEST(ImplicationTest, BasicTest) {}
+```
+
+* Each test can have multiple test methods defined in the comment part.
+
+* These can be simply defined with `/** Test methods */` or `/// Test methods` format. 
+
+* Test methods can be separated by commas or spaces.
+
+* Note: The `markers` should be exactly close to the test functions (without any empty lines)
```

Comment:
If this is a hard requirement, then please write "shall" instead of "should".
Otherwise, remove the sentence.